app/status/status.go

```diff
@@ -0,0 +1,123 @@
+// Cosi takes a file or a message and signs it collectively.
+// For usage, see README.md
+package main
+
+import (
+	"os"
+
+	"errors"
+	"time"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/app/lib/ui"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/sda"
+
+	"fmt"
+
+	"github.com/dedis/cothority/services/status"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+// RequestTimeOut defines when the client stops waiting for the CoSi group to
+// reply
+const RequestTimeOut = time.Second * 10
+
+const optionGroup = "group"
+const optionGroupShort = "g"
+
+func init() {
+	dbg.SetDebugVisible(1)
+	dbg.SetUseColors(false)
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Status"
+	app.Usage = "Get and print status of all servers in a file."
+	app.Commands = []cli.Command{
+		{
+			Name:    "getStatus",
```

Comment:
We need to change names here because it makes `./status getStatus`, what about `./status network` ?
Furthermore, this could be defined as the main command on the app so calling `./status` directly returns this information. It can still be kept also a sub command if needed.
what do you think ?